cmake/configs/default.cmake

```diff
@@ -219,6 +219,7 @@ hunter_config(libjson-rpc-cpp VERSION 0.7.0-p3)
 hunter_config(libmill VERSION 1.18)
 hunter_config(libogg VERSION 1.3.2-cmake3)
 hunter_config(libsodium VERSION 1.0.10)
+hunter_config(libthrift VERSION 0.10.0)
```

Comment:
Two equal projects with the different names?

Reply:
They pass in different cmake args. Also, because libthrift has no dependency on bison, it is cachable.

Reply:
Both use URL `https://github.com/hunter-packages/thrift/archive/v0.10.0-p2.tar.gz` - hence they are the same packages.

> They pass in different cmake args

Can be controlled by custom `CMAKE_ARGS`:
* https://github.com/ruslo/hunter/wiki/example.custom.config.id#custom-cmake-options

`CMAKE_ARGS` in `hunter.cmake` just set some reasonable default values. The rest is up to user. Also see this feature request: https://github.com/ruslo/hunter/issues/1106

Reply:
What about `hunter_cacheable` and the fact that libthrift supports many more platforms?

Reply:
Set best `CMAKE_ARGS` configuration as a default and add warnings to documentation about other options. If cacheable/uncacheable depends on CMAKE_ARGS then user can break it just by specifying custom options. Having two packages that do install same files is a badly designed solution.